roamance/src/components/profile/profile-header.tsx

```
@@ -0,0 +1,232 @@
+      }
+    } catch (error) {
+      console.error('Failed to update profile picture:', error);
+      toast.error('Failed to update profile picture');
+    } finally {
+      setIsUploading(false);
+    }
+  };
+
+  const handleUploadError = (error: Error) => {
+    console.error('Upload failed:', error);
+    toast.error('Image upload failed: ' + error.message);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)